CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Dradis Framework 4.1.0 (XXXX, 2021) ##
+
+*   Update HTML tag cleanup
```

Comment:
#weaksauce, particularly considering it's the same line as before (which was #weaksauce too!)

What about describing the improvements?